osdk/src/base_crate/x86_64.ld.template

```diff
@@ -58,6 +58,12 @@ SECTIONS
         __bss_end = .;
     }
 
+    .ex_table               : AT(ADDR(.ex_table) - KERNEL_VMA) {
```

Comment:
Seems that the reader of the linker script cannot tell what is "ex_table". Maybe you should add some comments to reference to the exception table implementation in the framework.